packages/render-fidelity-tools/src/common.ts

```diff
@@ -254,3 +262,7 @@ export class ImageComparator {
     return {analysis: {rmsDistanceRatio}};
   }
 }
+
+export function toDecibel(value: number): number {
+  return 10 * Math.log10(value);
+}
```

Comment:
Since this function is used in more than one script, i just refactored it into common.ts.  I also tried to write the fidelity test result as dB directly, but json file doesn't recognize Infinity and there's some trouble in parsing it, so i just stay with this way. If you have better idea on how to refactor this, please let me know, thanks!